Cargo.toml

```diff
@@ -31,3 +38,13 @@ features = [
     "Win32_UI_WindowsAndMessaging",
     "Win32_System_SystemServices",
 ]
+
```

Comment:
May you add the following here so that `docs.rs` tags items with the required features?
```toml
[package.metadata.docs.rs]
all-features = true
rustdoc-args = ["--cfg", "doc_cfg"]
```

Reply:
Yep, somehow I managed to forget this. I have changed the usages to `doc_cfg`